tsconfig.base.json

```diff
@@ -16,6 +16,6 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "strict": true,
-    "target": "ES2021"
+    "target": "ES2022"
```

Comment:
The change in this file was needed in order for TS to know about `Object.hasOwn`, which is what is used in ESLint.

However, we could revert this and switch to `Object.prototype.hasOwnProperty` if you think that'd be better.